databricks/koalas/tests/test_series_string.py

```diff
@@ -130,3 +137,182 @@ def test_string_isnumeric(self):
 
     def test_string_isdecimal(self):
         self.check_func(lambda x: x.str.isdecimal())
+
+    def test_string_cat(self):
+        ks = koalas.from_pandas(self.pds1)
+        with self.assertRaises(NotImplementedError):
+            ks.str.cat()
+
+    def test_string_center(self):
+        self.check_func(lambda x: x.str.center(0))
+        self.check_func(lambda x: x.str.center(10))
+        self.check_func(lambda x: x.str.center(10, 'x'))
+
+    def test_string_contains(self):
+        self.check_func(lambda x: x.str.contains('le', regex=False))
+        self.check_func(lambda x: x.str.contains('White', case=True, regex=False))
+        self.check_func(lambda x: x.str.contains('apples|carrots', regex=True))
+        self.check_func(lambda x: x.str.contains('BANANAS', flags=re.IGNORECASE, na=False))
+
+    def test_string_count(self):
+        self.check_func(lambda x: x.str.count('wh|Wh'))
+        self.check_func(lambda x: x.str.count('WH', flags=re.IGNORECASE))
+
+    def test_string_decode(self):
+        ks = koalas.from_pandas(self.pds1)
+        with self.assertRaises(NotImplementedError):
+            ks.str.decode('utf-8')
+
+    def test_string_encode(self):
+        ks = koalas.from_pandas(self.pds1)
+        with self.assertRaises(NotImplementedError):
+            ks.str.encode('utf-8')
+
+    def test_string_extract(self):
+        ks = koalas.from_pandas(self.pds1)
+        with self.assertRaises(NotImplementedError):
+            ks.str.extract('pat')
+
+    def test_string_extractall(self):
+        ks = koalas.from_pandas(self.pds1)
+        with self.assertRaises(NotImplementedError):
+            ks.str.extractall('pat')
+
+    def test_string_find(self):
+        self.check_func(lambda x: x.str.find('a'))
+        self.check_func(lambda x: x.str.find('a', start=3))
+        self.check_func(lambda x: x.str.find('a', start=0, end=1))
+
+    def test_string_findall(self):
+        self.check_func(lambda x: x.str.findall('es|as'))
+        self.check_func(lambda x: x.str.findall('wh.*', flags=re.IGNORECASE))
+
+    def test_string_index(self):
+        pds = pd.Series(['tea', 'eat'])
+        self.check_func_on_series(lambda x: x.str.index('ea'), pds)
+        with self.assertRaises(ValueError):
```

Comment:
I think it's fine to catch just `Exception`. Seems like lower Spark version throws a different exception.